healthcheck/utils.go

```diff
@@ -0,0 +1,19 @@
+package healthcheck
+
+import (
+	"math/rand"
+	"time"
+)
+
+// calcIntervalWithJitter returns a new duration based on a provided interval and a jitter percentage of 5%
+func calcIntervalWithJitter(interval time.Duration) time.Duration {
+	jitterPercentage := 0.05
```

Comment:
Magic number. Make this a constant.

Reply:
`jitterPercentage` is being stored in a variable which has a descriptive name, so I don't think it technically classifies as a magic number, but it should have been a `const` as pointed out, so have made it one now. Also noticed I made a mistake calling it a 'percentage' yet it is a decimal value have renamed into jitterAmount.